clang/test/SemaSYCL/reqd-work-group-size.cpp

```diff
@@ -96,17 +96,8 @@ void bar() {
     f32x32x1();
   });
 
-  // Support for reqd_work_group_size (and other SYCL attributes) that apply to
-  // lambda expressions is not implemented in clang yet.
-  // When it lands, the following code is expected to compile successfully.
-  //
-  // expected-error@+1 {{'reqd_work_group_size' attribute cannot be applied to types}}
-  kernel<class kernel_name9>([]() [[cl::reqd_work_group_size(32, 32, 32)]] {
```

Comment:
Maybe leave this as a positive test case?

Reply:
I believe codegen test case added is enough.

Reply:
I suggest to either keep it as a positive test, or remove all positive tests from the file.

Reply:
Done